### DISCUSSION TITLE
about querying the sample house imodel.

### DISCUSSION THREAD

**Discussion by ajay-toobler** [iTwin/itwinjs-core#4062] [Q&A]
https://www.youtube.com/watch?v=dZrhPBZXdqs&t=278s

in this video . I couldn't query like he did. 
### getting error of :

Failed to execute the ECSQL: ECClass **'DgnCustomItemTypes_HouseSchema.SmartDevice'** does not exist or could not be loaded.

### for this query :  
```
SELECT SmartDeviceId,
              Origin
              FROM DgnCustomItemTypes_HouseSchema.SmartDevice
              WHERE Origin IS NOT NULL
```

If u could help that would be great.

**Comment by aruniverse** [comment]
Hi! Could you try again with this model? https://github.com/iTwin/Course_1-Smart-House-Sample/blob/main/House_Model.dgn

**Reply by ajay-toobler** [reply]
yh i added and when i try to open that i modal console. it's showing
(I'm new to this. I'm currently understanding this by working on modal.)

**Error: Could not find a project with the specified criteria that the user has access to**

![image](https://user-images.githubusercontent.com/109501190/184063040-30401cbb-4137-4efe-82a5-ecdbfc3ca986.png)

**Reply by williamkbentley** [reply]
Looks like something went wrong when you created your iModel. How did you create it? The course wants you to use the [iTwin Synchronizer](https://www.youtube.com/watch?v=TBuYGyI1BL8)

**Reply by ajay-toobler** [reply]
is there a way to use itwin synchronizer / its alternative on ubuntu machine ?. when i used itwin synchronizer in windows. it worked fine. thanks for that too.

**Reply by aruniverse** [reply]
You could use our [Synchronization apis](https://developer.bentley.com/apis/synchronization/)
